File: 02_algorithm/programmers/Level1/나누어 떨어지는 숫자 배열.py
def solution(arr, divisor):
    answer = []
    for num in arr:
        if not num % divisor:
            answer.append(num)
    if len(answer) == 0:
        return [-1]
    return sorted(answer)

print(solution([5, 9, 7, 10], 5))
print(solution([2, 36, 1, 3], 1))
print(solution([3, 2, 6], 10))

File: 06_js/04_algorithm/programmers/Level1/나누어 떨어지는 숫자 배열.js
function solution(arr, divisor) {
    let select_num = [];
    for (let i = 0; i < arr.length; i++) {
        if (arr[i] % divisor === 0) {
            select_num.push(arr[i]);
        }
    }
    if (select_num.length === 0) {
        select_num.push(-1);
    }
    let answer = select_num.sort((a, b) => a - b );
    return answer;
}